Formula/imagemagick.rb

```diff
@@ -1,14 +1,13 @@
 class Imagemagick < Formula
   desc "Tools and libraries to manipulate images in many formats"
   homepage "https://imagemagick.org/index.php"
-  url "https://www.imagemagick.org/download/releases/ImageMagick-7.1.0-37.tar.xz"
-  sha256 "9a0cb5218ea687fad28eedf143dff9f0724eb62c73264231d690fe844ecce5ec"
+  url "https://imagemagick.org/archive/releases/ImageMagick-7.1.0-39.tar.xz"
```

Comment:
Not sure if only temporary issue with previous download URL. For now, just using other download URL from imagemagick.org.